CONTRIBUTING.md

```diff
@@ -42,10 +42,10 @@ To add a new page to the website, [create a new markdown file with YAML frontmat
 title: Timeturner
 
 # Category of the product (mandatory).
-# Possible values are os,db,app,lang,framework,device,service,server-app.
+# Possible values are os,database,app,lang,framework,device,service,server-app.
 # If you add a new value, please mention it in the PR Description. Some rough guidelines:
 # - os is for operating systems (and similar projects),
-# - db is for databases,
+# - database is for all kind of databases,
```

Comment:
Minor English language thing, might read better to pluralise "kinds"?  e.g.

 "database is for all kinds of databases"

Reply:
Thanks, applied.